ibis/expr/api.py

```diff
@@ -3988,6 +3988,23 @@ def _table_union(left, right, distinct=False):
     return op.to_expr()
 
 
+def _table_intersect(left, right):
+    """
+    Form the table set intersect of two table expressions having identical
+    schemas.
+
+    Parameters
+    ----------
+    right : TableExpr
```

Comment:
`left` parameter is missing.

 Not sure if in this case makes sense, but if it does, describing the parameters can be useful.

Reply:
@datapythonista I left this parameter out because it's not present in the union doc string either. I imagine that it's because in the api you can only do table1.union(table2) and table1.intersect(table2). I can still add it but we should then add it to union to be consistent

Reply:
yes pls do